website/content/en/docs/scorecard-alpha/_index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Scorecard (alpha)
+weight: 25
+---
+
+# operator-sdk scorecard (alpha)
+
+## Overview
+
+The scorecard command, part of the operator-sdk, executes tests 
+on your operator based upon a user defined configuration file 
+and test images.
+
+Tests are implemented within test images that are configured 
+and constructed to be executed by scorecard.
+
+Each test(s) is executed within a Pod by scorecard, pod logs 
+contain the test results, and scorecard aggregates the test 
+results to display to end users.  Scorecard therefore assumes 
+it is being executed with access to a configured Kube cluster.
+
+Scorecard has built-in basic and OLM tests but also provides a 
+means to execute custom user defined tests.
+
+## Requirements
+
+The scorecard tests make no assumptions as to the state of the 
+operator being tested.  Creating operators and custom resources 
+for an operator are left outside the scope of the scorecard itself.
+
+Scorecard tests can however create whatever resources they 
+require if the tests are designed for resource creation.
+
+Scorecard requires the following Kube environment:
+
+- Access to a Kubernetes v1.17.1+ cluster
+
+## Running the Scorecard
+
+1. Define a scorecard configuration file `config.yaml`.  A sample 
+configuration file [config.yaml][sample-config] is found within 
+the SDK repository. See [Config file](#config-file) for an explaination 
+of the configuration file format.  Unless you are executing custom 
+tests, you can just copy the provided example configuration file 
+into your project.
+2. Place the scorecard configuration file within your project 
+bundle directory a the following location `tests/scorecard/config.yaml`.  
+You can override the default location of the configuration file 
+by specifying the `--config` flag.
+3. Execute the [`scorecard` command][cli-scorecard]. See the 
+[Command args](#command-args) to check its options.
+
```

Comment:
May is better just replace the number by makerdown item list or * or -

Reply:
not sure, having numbers indicates a preferred order I think.

Reply:
I think that has a marker which will gen the numbers which make easier to update add new items. But It is fine.